cli/command/container/tty.go

```diff
@@ -34,19 +34,48 @@ func resizeTtyTo(ctx context.Context, client client.ContainerAPIClient, id strin
 	}
 
 	if err != nil {
-		logrus.Debugf("Error resize: %s", err)
+		logrus.Debugf("Error resize: %s\r", err)
```

Comment:
What is the reason for adding `\r` here?

Reply:
@kolyshkin Nice to see you again ^_^ 
Thanks for your review. It's because that:
```
root@iZ2zeesy3n96esegm7tue0Z:~# docker --debug exec -it k8s_nginxgateway_apigateway-p46ch_default_2de895e0-e8a8-11e8-badf-00163e0af11c_0 /bin/bash
DEBU[0000] Error resize: Error response from daemon: no such exec 
                                                                  root@apigateway-p46ch:/# ls
bin   dev  home  lib64  mnt  proc  run   srv  t    usr
boot  etc  lib   media  opt  root  sbin  sys  tmp  var
root@apigateway-p46ch:/#
```
When we use `--debug`, the first shell prompt don't return to the head of line?

Reply:
Nice catch! I've definitely seen it in the past.

Maybe add a comment why `\r` is there to the patch description, so anyone doing `git blame` can find the reason why.

Reply:
No code change, modify patch description now. Thanks.